tools/specs/lib/glbToI3dmSpec.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+var fs = require('fs');
+var Promise = require('bluebird');
+var extractI3dm = require('../../lib/extractI3dm');
+var glbToI3dm = require('../../lib/glbToI3dm');
+
+var fsReadFile = Promise.promisify(fs.readFile);
+
+var glbPath = './specs/data/CesiumTexturedBox/CesiumTexturedBox.glb';
+var i3dmPath = './specs/data/instancedWithTransform.i3dm';
+
+describe('glbToI3dm', function() {
+    var glbBuffer;
+    var featureTableJSONBuffer;
+    var featureTableBinaryBuffer;
+    beforeAll(function(done) {
+        fsReadFile(glbPath)
+            .then(function(data) {
+                glbBuffer = data;
+                var featureTable = {
+                    INSTANCES_LENGTH : 1,
+                    POSITION : {
+                        byteOffset : 0
+                    }
+                };
+                var featureTableString = JSON.stringify(featureTable);
+                featureTableJSONBuffer = Buffer.from(featureTableString);
+                featureTableBinaryBuffer = Buffer.alloc(12, 0); // [0, 0, 0]
```

Comment:
It would be best if the specs also used the `getBufferPadded` and `getJsonBufferPadded` functions.